README.md

```diff
@@ -6,20 +6,49 @@ Based on [Outer Wilds Korean Translation](https://outerwildsmods.com/mods/outerw
 
 ## Getting started
 
-- Create a base mod using the regular mod template.
-- Add ILocalizationAPI as an interface in your mod with the following code:
-```cs
-public interface ILocalizationAPI
-{
-    void RegisterLanguage(ModBehaviour mod, string name, string translationPath, string assetBundlePath, string fontPath, Func<string, string> fixer);
-    void RegisterLanguage(ModBehaviour mod, string name, string translationPath, string assetBundlePath, string fontPath);
-    void RegisterLanguage(ModBehaviour mod, string name, string translationPath);
-}
-```
+1. Create a base mod using [the regular mod template.](https://github.com/Raicuparta/ow-mod-template). 
+2. _Optional:_ Rename all files to match the mod name:
+   1. Instead of `ModTemplate/ModTemplate.cs`, rename files to `DothrakiTranslation/DothrakiTranslation.cs`. (This also applies to `ModTemplate.csproj.user`.)
+   2. Replace every instance of `ModTemplate` with `DothrakiTranslation` in the script texts (should be just the `ModTemplate.cs`).
+   3. Fix up `filename` in `manifest.json` accordingly.
+3. Add a dictionary entry to the end of the `manifest.json` dict that specifies the dependency for Outer Wilds Mod Manager:
+    ```json
+    "dependencies": [
+        "xen.LocalizationUtility"
+    ]
+    ```
 
-Access the utility mod in your mod code like so:
-```cs
-var api = ModHelper.Interaction.TryGetModApi<ILocalizationAPI>("xen.LocalizationUtility");
-```
+    Note that this will not automatically install the dependency _for you in development_ - you'll need to do so manually by downloading and enabling Interplanetary Polyglot in the Outer Wilds Mod Manager - but once you release the mod, it will be installed automatically for your users.
```

Comment:
It isn't actually installed automatically, but it does prompt them to install it themselves and won't let them play the game until they do